test/playwright/e2e/image-detail.spec.ts

```diff
@@ -37,7 +37,7 @@ test('shows the main image with its title as alt text', async ({ page }) => {
 
 test('does not show back to search results breadcrumb', async ({ page }) => {
   await goToCustomImagePage(page)
-  await expect(page.locator('text="Back to search results"')).not.toBeVisible({
+  await expect(page.locator('text="Back to results"')).not.toBeVisible({
```

Comment:
```suggestion
  await expect(
    page.locator(`text="${t('single-result.back')}"`)
  ).not.toBeVisible({
```

Reply:
Would be nice to use the translated version here and in the other two tests.

You'd also need to import `t`:
`import { t } from '~~/test/playwright/utils/navigation'`

Reply:
Why is the `t` located in `utils/navigation`? I looked for it but didn't expect the navigation util to have it.